keps/sig-testing/2539-continuously-deploy-k8s-prow/kep.yaml

```diff
@@ -0,0 +1,18 @@
+title: Continuously Deploy K8s Prow
+kep-number: 2539
+authors:
+  - "@chaodaiG"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-testing
+  - sig-release
+status: provisional
```

Comment:
Address the review comments and I am happy to merge a followup that bumps this to implementable